src/changelog/ticketChangelog.ts

```diff
@@ -0,0 +1,116 @@
+/**
+ * Implements methods for working with ticket scopes, with the ability to search
+ * for common scopes both within the project and within the tags
+ */
+
+import * as E from "fp-ts/Either";
+import * as O from "fp-ts/Option";
+import * as RA from "fp-ts/ReadonlyArray";
+import { pipe } from "fp-ts/lib/function";
+import { GenericTicket } from "../types";
+import { popFromRegex, isSameScope } from "../utils/validator";
+import { Scope, projectToScope, tagToScope } from "./types";
+
+export declare function warn(message: string): void;
+export declare function markdown(message: string): void;
+export declare function schedule<T>(asyncFunction: Promise<T>): void;
+
+// pattern used to recognize a scope label
+const regex = /(changelog-scope:|epic-)(.*)/m;
```

Comment:
You can avoid the comment if you give the variable a better name such as `scopeLabelRegex` 😃